Source/Core/PolygonOutlineGeometry.js

```diff
@@ -276,9 +276,9 @@ define([
      *       -75.0, 30.0,
      *       -70.0, 30.0,
      *       -68.0, 40.0
-     *     ])
-     *   },
-     *   extrudedHeight: 300000
+     *     ]),
```

Comment:
Looks like the branches got mixed up, and this reverts your previous fix.  Can you please update this?